deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -629,11 +630,24 @@ def merge_input_provider(index, item):
         input_inflation=input_inflation,
         input_average_record_size_bytes=input_average_record_size_bytes,
     )
-
     logger.info(
         f"partition-{params.source_partition_locator.partition_values},"
         f"compacted at: {params.last_stream_position_to_compact},"
     )
+    is_inplace_compacted: bool = (
+        params.source_partition_locator == params.destination_partition_locator
+    )
```

Comment:
There is a problem if two jobs run concurrently which is still not resolved. Say job 1 ran first where this check will yield true but for 2nd job, this could yield False which is result in incorrect RCF and eventually triggering rebases more often than required.